test/benchmark_utils/test_benchmark_utils.py

```diff
@@ -162,6 +162,16 @@ def test_timer(self):
         ).timeit(5).median
         self.assertIsInstance(sample, float)
 
+    @slowTest
+    def test_cpp_timer(self):
+        timer = benchmark_utils.Timer(
+            "torch::Tensor y = x + 1;",
+            setup="torch::Tensor x = torch::empty({1});",
+            language=benchmark_utils.Language.CPP,
+        )
+        t = timer.timeit(10)
```

Comment:
Is there really any good reason to run this for 10s? For the purpose of a test haha

Reply:
It's 10 iterations. `timer.blocked_autorange(min_run_time=10)` would be 10 seconds.

Reply:
Converse question, then, is why is this a slow test :) (compile time?)

Reply:
Yup, compile time.